project-base/src/Shopsys/ShopBundle/Resources/scripts/frontend/login.js

```diff
@@ -3,9 +3,30 @@
     Shopsys = window.Shopsys || {};
     Shopsys.login = Shopsys.login || {};
 
-    Shopsys.login.init = function () {
-        $('body').on('submit', '.js-front-login-window', function () {
-            $('.js-front-login-window-message').empty();
+    Shopsys.login.Login = function () {
+
+        this.init = function ($loginButton) {
+            $loginButton.click(showWindow);
+        };
+
+        function showWindow (event) {
+            Shopsys.ajax({
+                url: $(this).data('url'),
+                type: 'POST',
+                success: function (data) {
+                    var $window = Shopsys.window({
+                        content: data,
+                        textHeading: Shopsys.translator.trans('Login')
```

Comment:
The feature you're using here is not yet implemented (it belongs to the next commit).